src/content/docs/fr/tutorial/2-pages/4.mdx

```diff
@@ -0,0 +1,202 @@
+---
+type: tutorial
+title: Stylez votre page À propos
+description: |-
+  Tutoriel : Construisez votre premier blog Astro —
+  Ajoutez une balise de style Astro pour le stylage localisé sur la page
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Spoiler from '~/components/Spoiler.astro';
+import Box from '~/components/tutorial/Box.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+
+
+Maintenant que vous avez une page "À propos" avec du contenu vous concernant, stylisons-la !
+
+<PreCheck>
+  - Stylez des éléments sur une seule page
+  - Utilisez des variables CSS
+</PreCheck>
+
+
+## Stylez une page individuelle
+
+En utilisant les balises `<style></style>` d'Astro, vous pouvez styliser des éléments sur votre page. Ajouter des **attributs** et des **directives** à ces balises vous offre encore plus de possibilités de stylisation.
+
+1. Copiez le code suivant et collez-le dans `src/pages/about.astro` :
+
+    ```astro title="src/pages/about.astro" ins={6-11}
+    <html lang="fr">
+      <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width" />
+        <title>{pageTitle}</title>
+        <style>
+          h1 {
+            color: purple;
+            font-size: 4rem;
+          }
+        </style>
+      </head> 
+
+    ```
+
+    Vérifiez toutes les trois pages dans l'aperçu de votre navigateur.
+    
+    - De quelle couleur est le titre de la page de :
```

Comment:
I think, it's better to remove the `la page de` because then, we say `votre page`, which is a bit redundant.
```suggestion
    - De quelle couleur est le titre de :
```